client/ayon_houdini/plugins/publish/collect_local_render_instances.py

```diff
@@ -147,6 +147,8 @@ def process(self, instance):
                 "folderPath": instance.data["folderPath"],
                 "frameStart": instance.data["frameStartHandle"],
                 "frameEnd": instance.data["frameEndHandle"],
+                "frameStartHandle": instance.data["frameStartHandle"],
+                "frameEndHandle": instance.data["frameEndHandle"],
```

Comment:
Looking at this now, this should really be:

```suggestion
                "frameStart": instance.data["frameStart"],
                "frameEnd": instance.data["frameEnd"],
                "handleStart": instance.data["handleStart"],
                "handleEnd": instance.data["handleEnd"],
                "frameStartHandle": instance.data["frameStartHandle"],
                "frameEndHandle": instance.data["frameEndHandle"],
```
So that it has all the relevant frame attributes from the source instance that other plug-ins MAY use.

Reply:
Unfortunately, it may not work.
Because, it may fall to `wrong frame calculation with handles` bug again https://github.com/ynput/OpenPype/pull/5698

Reply:
anyways, if you think this PR is ready, feel free to click the merge button.

Reply:
Could you point me to where it would fail? But if those values would not work? Doesn't that mean that `instance.data` where we're copying it from is already wrong?

Reply:
@MustafaJafar Can you confirm whether it works for you if you ONLY set `frameStartHandle` and `frameEndHandle` and remove the `frameStart` and `frameEnd`?

Reply:
> Could you point me to where it would fail? But if those values would not work? Doesn't that mean that `instance.data` where we're copying it from is already wrong?

As far as I can remember, we are using the ROP node to get the full frame range.

> Can you confirm whether it works for you if you ONLY set frameStartHandle and frameEndHandle and remove the frameStart and frameEnd?

Let me test run it to refresh my memory.

Reply:
Removing `frameStart` and `frameEnd` in `aov_instance.data` there works perfect for me. And that'd align the instances also with what Collect ROP frame range plugin collects for other other instances. Because, Collect ROP frame range uses the `lib.get_frame_data` function. 

So making sure the instances have the same data (and then allowing other instances to do whatever they need should be good to go)

I SUSPECT that previously we had `frameStart` and `frameEnd` in there to AVOID the "asset handles" plug-in to trigger? The fix for _that_ would be to match the publish attributes of the original instances (so that we also 'copy' their decision on disable asset handles or not). Which would be this: https://github.com/ynput/ayon-houdini/blob/6586d16a0049c3e4e3c193f819cb844739817343/client/ayon_houdini/plugins/publish/collect_local_render_instances.py#L157

Reply:
@BigRoy it seems to me that some frame range data is not collected as I encountered this after testing with the suggestion. 
```
DEBUG: Looking for matching profile for: product_types: "render" | hosts: "houdini" | tasks: "art" | task_types: "Art"
DEBUG: "render" not found in "product_types": ['workfile']
DEBUG: "render" not found in "product_types": ['renderLayer', 'renderPass']
DEBUG: "render" not found in "product_types": ['review', 'workfile']
DEBUG: "houdini" not found in "hosts": ['aftereffects']
DEBUG: "render" not found in "product_types": ['staticMesh']
DEBUG: "render" not found in "product_types": ['skeletalMesh']
DEBUG: "render" not found in "product_types": ['hda']
DEBUG: Profile selected: {'product_types': ['render'], 'hosts': [], 'task_types': [], 'tasks': [], 'template': '{product[type]}{Task[name]}{Variant}'}
WARNING: Host's colorspace management is disabled.
Traceback (most recent call last):
  File "C:\Users\Kayla\AppData\Local\Ynput\AYON\dependency_packages\ayon_2402141620_windows.zip\dependencies\pyblish\plugin.py", line 527, in __explicit_process
    runner(*args)
  File "D:\ayon-addon_template\ayon-houdini\client\ayon_houdini\plugins\publish\collect_local_render_instances.py", line 148, in process
    "frameStart": instance.data["frameStart"],
KeyError: 'frameStart'
```

Reply:
@moonyuet  indeed we collect `frameStartHandle` and `frameEndHandle` from the rop node using this `lib.get_frame_data` [function](https://github.com/ynput/ayon-houdini/blob/62679b2f15fbc711bdbda9472fb9b9199355cb37/client/ayon_houdini/api/lib.py#L472-L523).

Then we use them to compute/use them directly for `frameStart` and `frameEnd`

Reply:
So, keeping only `frameStartHandle` and `frameEndHandle` works fine as the there's a late collector [collect_asset_handles.py](https://github.com/ynput/ayon-houdini/blob/develop/client/ayon_houdini/plugins/publish/collect_asset_handles.py) that should be compute 
- `frameStart`
- `framaEnd`
- `handleStart`
- `handleEnd`

```suggestion
                "frameStartHandle": instance.data["frameStartHandle"],
                "frameEndHandle": instance.data["frameEndHandle"],
```

oh my god, this also fixes a bug in the meta data
<table>
<tr>
<td>Before</td>
<td>After</td
</tr>
<tr>
<td><img src=https://github.com/user-attachments/assets/627a0d8f-658f-45dd-b5c9-62a56683f319></td>
<td><img src=https://github.com/user-attachments/assets/cce41378-67d9-4f23-a8b6-e40002f2ac08></td>
</tr>
</table>

Reply:
This should be resolved by 
https://github.com/ynput/ayon-houdini/pull/93/commits/93e27be30c5133a6d12b31fa0facda2c77717355
https://github.com/ynput/ayon-houdini/pull/93/commits/aa21abbb50c44b8c06b845eed961114e341cc88a

Feel free to un-resolve if it doesn't work for you.